devel/to-f90/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       compilers 1.0
+
+name            to-f90
+version         2000-03-21
```

Comment:
Isn't it 1999-03-08?

```
$ curl -IL http://cd.textfiles.com/simtel/simtel0101/simtel/fortran/to_f90.zip
HTTP/1.1 200 OK
Date: Tue, 27 Oct 2020 15:58:08 GMT
Server: Apache/2.4.37 (FreeBSD) OpenSSL/1.0.2o-freebsd
Last-Modified: Mon, 08 Mar 1999 08:00:00 GMT
ETag: "4517-345891547e000"
Accept-Ranges: bytes
Content-Length: 17687
Content-Type: application/zip
```

Reply:
You are right. History: I took the version number from my fink package, which I packaged long time ago. Presumably, it is the date, when I packaged it for fink the first time. At least, I found no better explanation. 

I would favour to leave it as it is, because setting it to 1999-03-08 makes macport looking outdated compared to fink.

Reply:
Or you could fix the version of the Fink package. Not sure that duplicating a Fink error is a good justification here.